DemoApp/ChatPresenter.swift

```diff
@@ -325,6 +319,17 @@ class DemoChatChannelListRouter: ChatChannelListRouter {
         ])
     }
 
+    /// *Only for testing purposes*. Please do not replicate this. When changing users, you should recreate the controller and the view controller
+    private func switchTo(user: UserCredentials) {
```

Comment:
I would completely remove switching users from demo app, tbh. If we have it just for crash testing purposes, it could be in a separate branch rebased on top of `develop`.

Reply:
I was inclined to do so, tbh. Let's remove it